client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { has } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import handlers from './wpcom';
+
+export const middleware = store => next => action =>
+	has( handlers, action.type )
```

Comment:
Micro-optimization (though, given this is a middleware run on every dispatch, seems warranted): [`has` does a lot of work](https://github.com/lodash/lodash/blob/4.17.2/lodash.js#L13129), specifically around converting a string path before testing the object. Could we just write this as `handlers.hasOwnProperty( action.type )` instead? Or if we want to get very micro* and accept implications of truthiness via prototype members, maybe just `handlers[ action.type ]`?

(* I think [this benchmark](https://jsperf.com/hasownproperty-vs-in-vs-undefined/12) had shown `hasOwnProperty` being slower, though not loading for me at the moment)